gen/suites/vm_violations/caller_validation.go

```diff
@@ -5,21 +5,19 @@ import (
 	"github.com/filecoin-project/go-state-types/abi"
 	"github.com/filecoin-project/go-state-types/big"
 	"github.com/filecoin-project/go-state-types/exitcode"
-	typegen "github.com/whyrusleeping/cbor-gen"
-
 	"github.com/filecoin-project/lotus/conformance/chaos"
 	. "github.com/filecoin-project/test-vectors/gen/builders"
 )
 
-func callerValidation(branch chaos.CallerValidationBranch, expectedCode exitcode.ExitCode) func(v *MessageVectorBuilder) {
+func callerValidation(buildArgs func() chaos.CallerValidationArgs, expectedCode exitcode.ExitCode) func(v *MessageVectorBuilder) {
```

Comment:
why do you need to defer buildArgs computation? none of the cases in main seem to reference something where the branch can't be passed directly.

Reply:
artifact of the PR this came from where it was needed - it's not now so I'll get this fixed